src/style/buttons.module.css

```diff
@@ -13,8 +13,19 @@
     background-color: var(--ALLEN_LIGHT_10) !important;
 }
 
+.dark-blue-hover-button {
+    background-color: transparent;
+    color: var(--primary-color) !important;
+    border: 1px solid var(--primary-color);
+}
+
+.dark-blue-hover-button:hover {
+    color: var(--WHITE) !important;
+    background-color: var(--primary-color) !important;
+}
+
```

Comment:
theses `!important` tags are used a lot in the repo, do you remember what is making these properties hard to target?